src/components/ConfirmationModal.vue

```diff
@@ -67,11 +67,31 @@ export default {
 
 <style lang="scss" scoped>
 .confirm-modal {
-	padding: 20px;
+	padding: 24px 32px;
+
+	h2 {
+		margin-bottom: 20px;
+		font-size: 24px;
+		font-weight: 600;
+	}
 
 	&__buttons {
 		display: flex;
-		justify-content: space-between;
+		justify-content: flex-end;
+		gap: 16px;
+		margin-top: 30px;
+		padding-top: 20px;
+
```

Comment:
this part is correct, you want to add either a margin or a padding, not both as they both add space. 

The buttons should have space between so they are aligned left and right. So keep that as it is.